### PR TITLE
[HOLD] Add license

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Added license to address https://github.com/blackbaud/skyux2-docs/issues/894

### CODE REVIEW

**Comment by Blackbaud-BenLambert** [comment]
misc question - do we need to make this same change in other skyux templates?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (blackbaud-johnly) [inline LICENSE:1]: I think this should be `The MIT License (MIT)`.

**Comment by blackbaud-johnly** [comment]
@Blackbaud-BenLambert, we're going through and adding the license to all the repos where it's missing. This should include all the templates as far as I know.

**Comment by Blackbaud-DeniseCisneros** [comment]
@Blackbaud-BenLambert good question! @Blackbaud-BobbyEarl made a good point and asked if template repos should include licenses, and if they are included, should they be removed when running `skyux new`. I plan to talk that through with the team tomorrow, and I'll update with that decision after. Until then, placing this PR on hold.

**Comment by Blackbaud-DeniseCisneros** [comment]
After chatting with the SKY UX team, they decided to omit the license from templates. Closing this as a result.

**Pull request closed without merging**